plugin/pkg/scheduler/factory/factory.go

```diff
@@ -211,6 +223,13 @@ func NewConfigFactory(
 					if err := c.podQueue.Delete(obj.(*v1.Pod)); err != nil {
 						runtime.HandleError(fmt.Errorf("unable to dequeue %T: %v", obj, err))
 					}
+					pod, ok := obj.(*v1.Pod)
```

Comment:
please move this line to the beginning of this function and don't add "ok" and don't check it. We prefer to see a panic if the obj is not a pod. See `UpdateFunc` as an example.

Reply:
done